src/connectors/answers/connectAnswers.ts

```diff
@@ -0,0 +1,217 @@
+import escapeHits from '../../lib/escape-highlight';
+import {
+  checkRendering,
+  createDocumentationMessageGenerator,
+  createConcurrentSafePromise,
+  addQueryID,
+  debounce,
+  addAbsolutePosition,
+  noop,
+} from '../../lib/utils';
```

Comment:
```suggestion
import {
  checkRendering,
  createDocumentationMessageGenerator,
  createConcurrentSafePromise,
  addQueryID,
  debounce,
  addAbsolutePosition,
  noop,
  escapeHits,
} from '../../lib/utils';
```

After merging master, this will be needed